.github/workflows/code-health.yml

```diff
@@ -30,5 +30,192 @@ jobs:
         run: yarn install --check-files
       - name: build
         run: npx projen build
-
-
+      - name: Find mutations
```

Comment:
Discussed in standup - lets split the release test out into a `package.yml` workflow. Code health should be compile tests & validation, whereas ensuring the release is always possible is a packaging workflow.

Reply:
I created a package.yml which runs only if the code-health is successful. See [GitHub Action summary](https://github.com/mongodb/awscdk-resources-mongodbatlas/actions/runs/4765583761)


![Screenshot 2023-04-21 at 14 56 08](https://user-images.githubusercontent.com/5663078/233654582-a0268f0e-ff5b-4914-9345-58ef0a52b94a.png)